scripts/run_single_node.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -e
+
+# This script starts a single node running the local network with staking disabled and expects the caller to take care of cleaning up the created AvalancheGo process.
+# This script uses the data directory to use node configuration and populate data into a predictable location.
+if ! [[ "$0" =~ scripts/run_single_node.sh ]]; then
+  echo "must be run from repository root, but got $0"
+  exit 255
+fi
+
+# Load the versions
+SUBNET_EVM_PATH=$(
+  cd "$(dirname "${BASH_SOURCE[0]}")"
+  cd .. && pwd
+)
+source "$SUBNET_EVM_PATH"/scripts/versions.sh
+
+# Load the constants
+source "$SUBNET_EVM_PATH"/scripts/constants.sh
+
+# Set up avalanche binary path and assume build directory is set
+AVALANCHE_BINARY_PATH=${AVALANCHE_BINARY_PATH:-"$GOPATH/src/github.com/ava-labs/avalanchego/build/avalanchego"}
+PLUGIN_DIR=${PLUGIN_DIR:-"$GOPATH/src/github.com/ava-labs/avalanchego/build/plugins"}
```

Comment:
Actually, the capability is already available through this run script, so we should document this in `./scripts/run_ginkgo.sh`